app/api/server_test.go

```diff
@@ -0,0 +1,79 @@
+package api
+
+import (
+	"encoding/xml"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/umputun/feed-master/app/config"
```

Comment:
those imports are not grouped properly. Not sure what IDE/edition you are using but either way `gomimport` or some similar formatter should be avail and will help with this